usr/src/cmd/zpool/zpool_main.c

```diff
@@ -652,11 +665,40 @@ zpool_do_labelclear(int argc, char **argv)
 	pool_state_t state;
 	boolean_t inuse = B_FALSE;
 	boolean_t force = B_FALSE;
+	boolean_t force_invalid = B_FALSE;
+	boolean_t wipe = B_FALSE;
+	unsigned int start = 0, n = VDEV_LABELS;
+	long long index = 0;
+	const char *errstr;
 
 	/* check options */
-	while ((c = getopt(argc, argv, "f")) != -1) {
+	while ((c = getopt(argc, argv, "bei:wf")) != -1) {
 		switch (c) {
+		case 'b':
+			start = 0;
+			n = VDEV_LABELS / 2;
+			break;
+		case 'e':
+			start = VDEV_LABELS / 2;
+			n = VDEV_LABELS / 2;
+			break;
+		case 'i':
+			index = strtonum(optarg, 0, VDEV_LABELS - 1, &errstr);
+			if(errstr) {
```

Comment:
cstyle: add space after `if`, and add explicit `!= NULL`:
`if (strerr != NULL) {`